docs/best-practices/folders-structure.md

```diff
@@ -80,3 +90,39 @@ The main difference between a KMM and a native project is that the presentation
             │   └── model
             └── FeatureProvider.kt # DI Component. Provides interactor mostly
 ```
+</TabItem>
+
+<TabItem value="angular">
+
+### Angular Project structure
+
+```txt
+.
+└── project
+    └── src
+        ├── app                      # Angular stuff
+        │   ├── feature-name         # UI module
+        │   │   ├── component-name   # Includes component + html + css + view-state + test
+        │   │   ├── feature.module.ts
+        │   │   └── feature-routing.module.ts
+        │   ├── shared               # A folder to place services, pipes and every cross-feature class
```

Comment:
We would need to review Angular best practices. I think the singletons should go into a `core` module, or directly add them to the `app.module.ts`… but I've never really understood the pattern, so I don't know.

Reply:
https://aglowiditsolutions.com/blog/angular-best-practices/

This makes me think that pipes and directives are ok here, but services are feature related, taking into account that they are not containing http connections, only observables or shared data, so they should be into ui-feature. Wdyt?

Reply:
And about the core folder shown in the article... it does not convince me much. footer and header should be into shared components, authentication and others... are folders that exist in this structure because their not using clean architecture.